src/System.Numerics.Tensors/Directory.Build.props

```diff
@@ -4,5 +4,7 @@
     <PackageVersion>0.3.0</PackageVersion>
```

Comment:
Technically we don't need to bump minor as part of 5.0 since we're not shipping a 0.2.0 version, but not doing so would make us go backward in package versions from the previewN ones we already shipped.